crates/rustc_codegen_spirv/src/spirv_type_constraints.rs

```diff
@@ -945,6 +953,182 @@ pub fn instruction_signatures(op: Op) -> Option<&'static [InstSig<'static>]> {
         | Op::SubgroupAvcSicGetInterRawSadsINTEL => {
             reserved!(SPV_INTEL_device_side_avc_motion_estimation);
         }
+        // SPV_EXT_mesh_shader
+        Op::EmitMeshTasksEXT | Op::SetMeshOutputsEXT => {
+            // NOTE(eddyb) we actually use these despite not being in the standard yet.
+            // reserved!(SPV_EXT_mesh_shader)
```

Comment:
Should this be uncommented?

Reply:
Removed the comment and code. Previously, that was `reserved!()` aka. not implemented and now it should just do nothing when validating type constraints on these intrinsics, as they're already validated by the functions representing these intrinsics (and so do most other intrinsics as well).

Reply:
And rebased.